fresh build fails on imagemin-gifsicle module

1. clone wsk
1. install npm modules, gulp
1. run `gulp`
1. error:

```sh
~/code/web-starter-kit on master
$ gulp
[00:25:39] Using gulpfile ~/code/web-starter-kit/gulpfile.js
[00:25:39] Starting 'clean'...
[00:25:39] Finished 'clean' after 6.62 ms
[00:25:39] Starting 'default'...
[00:25:39] Starting 'styles'...
[00:25:42] 'styles' all files 93.17 kB
[00:25:42] Finished 'styles' after 3.05 s
[00:25:42] Starting 'jshint'...
[00:25:42] Starting 'html'...
[00:25:43] Starting 'images'...
[00:25:43] 'images' errored after 345 ms
[00:25:43] Error: Cannot find module 'imagemin-gifsicle'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/paulirish/code/web-starter-kit/node_modules/gulp-imagemin/node_modules/imagemin/index.js:117:27)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
[00:25:43] 'default' errored after 4.75 s
[00:25:43] Error: [object Object]
    at formatError (/Users/paulirish/.homebrew/lib/node_modules/gulp/bin/gulp.js:161:10)
    at Gulp.<anonymous> (/Users/paulirish/.homebrew/lib/node_modules/gulp/bin/gulp.js:187:15)
    at Gulp.emit (events.js:117:20)
    at Gulp.Orchestrator._emitTaskDone (/Users/paulirish/code/web-starter-kit/node_modules/gulp/node_modules/orchestrator/index.js:264:8)
    at /Users/paulirish/code/web-starter-kit/node_modules/gulp/node_modules/orchestrator/index.js:275:23
    at finish (/Users/paulirish/code/web-starter-kit/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:21:8)
    at cb (/Users/paulirish/code/web-starter-kit/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:29:3)
    at finish (/Users/paulirish/code/web-starter-kit/node_modules/run-sequence/index.js:45:5)
    at Gulp.onError (/Users/paulirish/code/web-starter-kit/node_modules/run-sequence/index.js:52:4)
    at Gulp.emit (events.js:117:20)
[00:25:43] Finished 'jshint' after 1.76 s
[00:25:44] 'html' all files 44.17 kB
[00:25:44] Finished 'html' after 1.54 s
~/code/web-starter-kit on master
```

npm really doesn't want to successfully install that optional dep:

```sh
~/code/web-starter-kit/node_modules/gulp-imagemin/node_modules/imagemin on master
$ npm install imagemin-gifsicle
npm WARN optional dep failed, continuing imagemin-gifsicle@3.0.0
$ cd ~/code/web-starter-kit/node_modules/gulp-imagemin/node_modules/
~/code/web-starter-kit/node_modules/gulp-imagemin/node_modules on master
$ ls
chalk         each-async    gulp-util     imagemin      object-assign pretty-bytes  through2
```

am i being noob or what

--- a/index.js
+++ b/index.js
@@ -3,7 +3,7 @@
  * Module dependencies.
  */
 
-var max = require('max-component');
+var longest = require('longest');
 var fmt = require('util').format;
 var assert = require('assert');
 
@@ -92,7 +92,7 @@ Logger.prototype.__log__ = function(type, color, args){
  */
 
 Logger.prototype.padleft = function(type){
-  var len = max(this.types, '.length');
+  var len = longest(this.types).length;
   return Array(this.indent + 1 + len - type.length).join(' ');
 };
 

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "test": "make test"
   },
   "dependencies": {
-    "max-component": "^1.0.0"
+    "longest": "^0.2.1"
   },
   "devDependencies": {
     "better-assert": "^1.0.1",

